src/ResourceManagement/NotificationHubs/NotificationHubs.Tests/NotificationHubs.Tests.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
We do not need a separate solution for tests. One solution should contain the code and test project. So please remove it.